### PR TITLE
[MFTF] Set correct url for CmsPageEditPage

### PR DESCRIPTION
**Pull request by Usik2203**
### Description (*)
This PR set correct url for `CmsPageEditPage`

### Related Pull Requests
https://github.com/magento/partners-magento2ee/pull/176

### CODE REVIEW

File: app/code/Magento/Cms/Test/Mftf/Page/CmsPageEditPage.xml [hunk 1]
```diff
@@ -7,7 +7,7 @@
  -->
 <pages xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xsi:noNamespaceSchemaLocation="urn:magento:mftf:Page/etc/PageObject.xsd">
-    <page name="CmsPageEditPage" area="admin" url="admin/cms_page/edit/page_id/{{var}}" parameterized="true" module="Magento_Cms">
+    <page name="CmsPageEditPage" area="admin" url="cms/page/edit/page_id/{{var}}" parameterized="true" module="Magento_Cms">
```
Comment (lbajsarowicz) [inline app/code/Magento/Cms/Test/Mftf/Page/CmsPageEditPage.xml]: Could you, by the way, change this generic `{{var}}` to something more meaningful?
```suggestion
    <page name="CmsPageEditPage" area="admin" url="cms/page/edit/page_id/{{pageId}}" parameterized="true" module="Magento_Cms">
```

**Review by lbajsarowicz — CHANGES_REQUESTED** [review]
We also should proceed with moving towards `{Area}` + `{Page}` + Page suffix.
If you have some spare time you can rename the Page and it's references to:
`AdminCmsPageEditPage`.

The most important for now is just to get rid of generic parameter names.

**Comment by lbajsarowicz** [comment]
I think that previous URL was correct:
image

In my example you have:
`/admin/` -> backend path
`/admin/cms/...` URL you try to change

**Comment by Usik2203** [comment]
@magento give me 2.4-develop instance

**Comment by magento-engcom-team** [comment]
Hi @Usik2203. Thank you for your request. I'm working on Magento 2.4-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @Usik2203, here is your [Magento instance](https://i-27563-2-4-develop.instances.magento-community.engineering).
Admin access: [https://i-27563-2-4-develop.instances.magento-community.engineering/admin_4985](https://i-27563-2-4-develop.instances.magento-community.engineering/admin_4985)
Login: `52b7282e` Password: `204ba286b5a3`
Instance will be terminated in up to 3 hours.

**Comment by lbajsarowicz** [comment]
@magento give me 2.3.4 instance

**Comment by magento-engcom-team** [comment]
Hi @lbajsarowicz. Thank you for your request. I'm working on Magento 2.3.4 instance for you

**Comment by magento-engcom-team** [comment]
Hi @lbajsarowicz, here is your [Magento instance](https://i-27563-2-3-4.instances.magento-community.engineering).
Admin access: [https://i-27563-2-3-4.instances.magento-community.engineering/admin_d4a6](https://i-27563-2-3-4.instances.magento-community.engineering/admin_d4a6)
Login: `ec821e26` Password: `51926d76550a`
Instance will be terminated in up to 3 hours.

**Comment by lbajsarowicz** [comment]
OK. I was wrong. For some reason my local env prefixed that with `/admin/`. 
So only replace the generic parameter name and we're good.

**Review by lbajsarowicz — APPROVED** [review]
Thank you! Well done.
We need to wait for CI, to check if some PageBuilder / EE / B2B are not failing.

**Comment by magento-engcom-team** [comment]
Hi @lbajsarowicz, thank you for the review.
[ENGCOM-7255](https://jira.corp.magento.com/browse/ENGCOM-7255) has been created to process this Pull Request
:eight_spoked_asterisk: @lbajsarowicz, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Comment by Usik2203** [comment]
@lbajsarowicz Thanks for review this PR and recommendations.

**Comment by engcom-Echo** [comment]
@magento run Functional Tests EE, Functional Tests B2B

**Comment by engcom-Delta** [comment]
:heavy_check_mark: QA passed

**Comment by engcom-Echo** [comment]
@magento run all tests

**Comment by engcom-Echo** [comment]
@magento run all tests

**Comment by engcom-Echo** [comment]
Failed functional tests not related to the changes in this PR

**Pull request merged**